api/nakadi-event-bus-api.yaml

```diff
@@ -1550,6 +1608,18 @@ definitions:
           contains at least one of the scopes in this list.
           If no scopes provided then anyone can consume from this event type.
 
+      created_at:
+        type: string
+        pattern: date-time
+        description: |
+          Date and time when this event type was created.
+
+      updated_at:
```

Comment:
:+1: to the date - is the name `updated_at` consistent with the rest of the API?

(gr)